### DISCUSSION TITLE
moving existing beets-managed directories

### DISCUSSION THREAD

**Discussion by jmgant** [beetbox/beets#4323] [General]
I made a mistake and placed an extra space in my format path. now, I want to fix it. how can I:
a) fix the directories manually while letting beets know of my changes (not metadata, just file paths)?
or
b)after fixing the config, have beets rename the directories?

it's my understanding that if I re-import an existing collection, beets won't make any changes. I want it to change things in this case, but just file/directory names, not tags. 

so far, I've considered making a new library path and use the -A flag with move:yes in the config, and running import against the old library.

**Comment by sampsyo** [comment]
Hi! Have you see [the relevant FAQ answer](https://beets.readthedocs.io/en/stable/faq.html#rename-my-files-according-to-a-new-path-format-configuration)?
